server/neptune/workflows/internal/pr/revision/state.go

```diff
@@ -30,7 +35,15 @@ func (s *StateReceiver) Receive(ctx workflow.Context, c workflow.ReceiveChannel,
 }
 
 func (s *StateReceiver) Notify(ctx workflow.Context, workflowState *state.Workflow, roots map[string]RootInfo) {
-	rootInfo := roots[workflowState.ID]
+	rootInfo, ok := roots[workflowState.ID]
```

Comment:
could we add a quick comment on what scenarios we may be protecting against here?

Reply:
done